testkit/src/main/scala/org/bitcoins/testkit/wallet/DLCWalletUtil.scala

```diff
@@ -101,4 +107,45 @@ object DLCWalletUtil {
 
   lazy val sampleDLCSign: DLCSign =
     DLCSign(dummyCETSigs, dummyFundingSignatures, sampleDLCEventId)
+
+  def initDLC(fundedWalletA: FundedWallet, fundedWalletB: FundedWallet)(
```

Comment:
`setupDLC` is probably a better name as we use `initDLC` to mean something else internally. Also a scaladoc would be nice here